instant-xml/src/lib.rs

```diff
@@ -5,12 +5,31 @@ use thiserror::Error;
 pub use xmlparser;
 
 pub use macros::{FromXml, ToXml};
+use parse::XmlParser;
 
+pub mod impls;
 #[doc(hidden)]
 pub mod parse;
 
+pub struct TagData {
```

Comment:
Pretty sure all the members here could be `&str` instead of `String`?

Reply:
Done